pywps/app/Process.py

```diff
@@ -313,6 +315,7 @@ def launch_next_process(self):
             process._set_uuid(uuid)
             process._setup_status_storage()
             process.async_ = True
+            process.setup_outputs_from_wps_request(new_wps_request)
```

Comment:
This line is the main change in the PR. The rest is just creating a method for this code.